### PR TITLE
Clean up helm charts

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

N/A

### Description of the Change

Make the helm chart look more like all other helm charts online
and add a README for the parameters in ETOS.

### Alternate Designs

We could've kept it as is, but I think it's better to try and make these charts as similar to other charts as possible.

### Benefits

Easier for users to understand what is going on.

### Possible Drawbacks

This is a major change

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

**Review by andjoe-axis — APPROVED** [review]
+1

**Review by fredjn — APPROVED** [review]

File: charts/etos/charts/api/values.yaml [hunk 1]
```diff
@@ -15,12 +16,33 @@ fullnameOverride: ""
...
+  config:
+    filebeat.yml: |-
+      filebeat.inputs:
+      - type: log
+        paths:
+          /usr/share/filebeat/logs/*
+      processors:
+        - add_host_metadata:
+      output.logstash:
+        hosts: ["logstash-logstash.elasticsearch.svc.cluster.local:5044"]configMap: "etos"
```
Comment (t-persson) [inline charts/etos/charts/api/values.yaml:45]:
```suggestion
        hosts: ["logstash-logstash.elasticsearch.svc.cluster.local:5044"]
```

File: charts/etos/values.yaml [hunk 2]
```diff
@@ -82,8 +17,134 @@ serviceAccount:
...
+api:
+  service:
+    port: 80
+    name: "etos-api"
+
+# Suite runner configuration. Suite runner is not a subchart to this repository
+# but tried to make it look like it is.
+suiteRunner:
+  image: "registry.nordix.org/eiffel/etos-suite-runner"
+  tag: "2.0.0"
```
Comment (t-persson) [inline charts/etos/values.yaml:34]:
```suggestion
  image:
      repository: "registry.nordix.org/eiffel/etos-suite-runner"
      tag: "2.0.0"
```

Comment (t-persson) [reply charts/etos/values.yaml:34]: Requires a change in configmap

File: charts/etos/templates/configmap.yaml [hunk 3]
```diff
@@ -1,57 +1,41 @@
...
-  ETOS_SIDECAR_ENABLED: {{ .Values.global.filebeatSidecar.enabled | quote }}
-  ETOS_SIDECAR_IMAGE: {{ .Values.global.filebeatSidecar.image }}
+  SUITE_RUNNER: {{ .Values.suiteRunner.image }}:{{ .Values.suiteRunner.tag }}
 
-  ETR_VERSION: {{ .Values.testRunner.version | quote }}
+  ETOS_GRAPHQL_SERVER: {{ required "ETOS requires an Eiffel GraphQL API instance`" .Values.graphql.host }}
+  ETOS_ENVIRONMENT_PROVIDER: {{ required "ETOS requires the environment provider host to be set" .Values.hosts.environmentProvider }}
+  ETOS_API: {{ required "ETOS requires the API host to be set" .Values.hosts.API }}
+
+  {{- if .Values.suiteRunner.filebeat.eneabled }}
```
Comment (t-persson) [inline charts/etos/templates/configmap.yaml:16]:
```suggestion
  {{- if .Values.suiteRunner.filebeat.enabled }}
```

File: charts/etos/templates/configmap.yaml [hunk 4]
```diff
@@ -1,57 +1,41 @@
...
+  name: {{ .Values.configMap }}
+  labels:
+    {{- include "etos.labels" . | nindent 4 }}
 data:
   DEV: {{ .Values.global.development | quote }}
 
-  {{- include "etos.suiteRunnerContainerImage" . | indent 2 }}
-  ETOS_SIDECAR_ENABLED: {{ .Values.global.filebeatSidecar.enabled | quote }}
-  ETOS_SIDECAR_IMAGE: {{ .Values.global.filebeatSidecar.image }}
+  SUITE_RUNNER: {{ .Values.suiteRunner.image }}:{{ .Values.suiteRunner.tag }}
```
Comment (t-persson) [inline charts/etos/templates/configmap.yaml:10]:
```suggestion
  SUITE_RUNNER: {{ .Values.suiteRunner.image.repository }}:{{ .Values.suiteRunner.image.tag }}
```

**Pull request merged**